src/plugins/newGuildSettings/index.tsx

```diff
@@ -91,25 +133,5 @@ export default definePlugin({
         }
     ],
     settings,
-
-    handleMute(guildId: string | null) {
-        if (guildId === "@me" || guildId === "null" || guildId == null) return;
-        updateGuildNotificationSettings(guildId,
-            {
-                muted: settings.store.guild,
-                suppress_everyone: settings.store.everyone,
-                suppress_roles: settings.store.role,
-                mute_scheduled_events: settings.store.events,
-                notify_highlights: settings.store.highlights ? 1 : 0
-            });
-        if (settings.store.messages !== 3) {
-            updateGuildNotificationSettings(guildId,
-                {
-                    message_notifications: settings.store.messages,
-                });
-        }
-        if (settings.store.showAllChannels && isOptInEnabledForGuild(guildId)) {
-            toggleShowAllChannels(guildId);
-        }
-    }
+    handleMute: handleSettings
```

Comment:
while we're already at it, let's fix the outdated name! (also have to do this inside patches)
```suggestion
    applyDefaultSettings
```